tests/publiccloud/check_registercloudguest.pm

```diff
@@ -45,7 +44,7 @@ sub run {
         # CHOST images don't have registercloudguest pre-installed. To install it we need to register which make it impossible to do
         # all BYOS related checks. So we just regestering system and going further
         registercloudguest($instance);
-    } elsif (is_byos()) {
+    } elsif (is_byos() && !check_var('PUBLIC_CLOUD_CHECK_CLOUDREGISTER_EXECUTED', '1')) {
```

Comment:
so if we run this test in latest variable is not defined it will get default value 1 :thinking:  so we won't execute what we need to , right ?

Reply:
I don't understand.

Reply:
brain misfunction . ignore it